### PR TITLE
OCPBUGS-24218: Annotate serviceaccount-ca with ownership information

### PR DESCRIPTION
**Pull request by vrutkovs**

### CODE REVIEW

File: pkg/operator/targetconfigcontroller/targetconfigcontroller.go [hunk 1]
```diff
@@ -335,6 +335,8 @@ func manageServiceAccountCABundle(ctx context.Context, lister corev1listers.Conf
 	requiredConfigMap, err := resourcesynccontroller.CombineCABundleConfigMaps(
 		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.TargetNamespace, Name: "serviceaccount-ca"},
 		lister,
+		"kube-scheduler",
+		"",
```
Comment (deads2k) [inline pkg/operator/targetconfigcontroller/targetconfigcontroller.go:339]: leave for another PR, but I really look forward to learning what this is for.  I don't remember it.

**Comment by ingvagabund** [comment]
@vrutkovs would you please rebase the PR to reduce the changes? The library-go deps were updated ~30 minutes ago.

**Comment by vrutkovs** [comment]
Sure, done
